analysis_options.yaml

```diff
@@ -1,8 +1,6 @@
 include: package:lints/recommended.yaml
 
 analyzer:
-  strong-mode:
-    implicit-casts: false
```

Comment:
if you want you can switch to:

```yaml
analyzer:
  language:
    strict-casts: true
```

Should be nearly the same

Reply:
Oh!  I do.  And fancy that, there's even a quick fix!  😎